packages/legend-application-studio/src/stores/LegendStudioBaseStore.ts

```diff
@@ -182,12 +182,14 @@ export class LegendStudioBaseStore {
       this.isSDLCAuthorized =
         (yield this.sdlcServerClient.isAuthorized()) as boolean;
       if (!this.isSDLCAuthorized) {
-        this.applicationStore.navigationService.navigator.goToAddress(
-          SDLCServerClient.authorizeCallbackUrl(
-            this.applicationStore.config.sdlcServerUrl,
-            this.applicationStore.navigationService.navigator.getCurrentAddress(),
-          ),
-        );
+        yield new Promise(() => {
```

Comment:
I don't think this wrapping of promise is doing anything, I wonder what are we trying to do here really?

Do we want to have a timeout/a wait? if so we can do something similar to a Thread.sleep - i.e. await setTimeout... in JS

Since we're going to a new page anyway and that's end of life of the current app, should we just simply call `return;` right here?